src/js/components/Hello.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+
+class Hello extends Component {
+
+  	constructor() {
+    	super();
+  	}
+
+  	render() {
+    	return (
+      		<h1 id="hello-react-para">hello react!</h1>
+    	);
+  	}
+
+}
+
+export default Hello;
+
+ReactDOM.render(<Hello />, document.getElementById("hello-react"));
```

Comment:
this works but depending on loading it's possible it won't work, put it in a place that ensures it will be initialized properly